src/components/SearchBar/SearchBar.stories.tsx

```diff
@@ -0,0 +1,52 @@
+import {StoryObj} from '@storybook/react-native';
+import {Decorator} from '../../../.storybook/decorators';
```

Comment:
Can we add decorators to index file in storybook and import from there?

Reply:
I tried exporting it though index.ts file and it is throwing me error. so i reverted it.

Reply:
Prajna, I have resolved the import issue of the decorator. But i am getting this cyclic dependency warning. Apart from that it is working fine as expected.

`Require cycle: .storybook/index.ts -> .storybook/storybook.requires.js -> src/screens/HomeScreen/components/SearchCarousel/SearchCarousel.stories.tsx -> .storybook/index.ts`.